tests/test-make-relative-remote/test.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
```

Comment:
Try removing this commit, delete tests/ structure and integration test and add the previous commit on a graveyward.md for future reference.